components/locale/Switcher.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+  const { locale, locales } = useI18n()
+  // generates URLs to switch between locales
+  const switchLocalePath = useSwitchLocalePath()
+  const availableLocales = computed(() => {
+    // returns a list of locales, exluding current locale
+    return locales.value.filter((i) => i.code !== locale.value)
+  })
+  const currentLocale = computed(() => {
+    return locales.value.filter((i) => i.code === locale.value)
+  })
+</script>
+
+<template>
+  <VMenu transition="scale-transition">
```

Comment:
I think we should change to `slide-y-transition`, both for this component and the `UserMenu` component. You can change both in this PR.

Reply:
should be fixed